code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -527,9 +527,11 @@
 		"r_foot" = list("path" = /obj/item/organ/external/foot/right/ipc)
 		)
 
-	abilities = list(/mob/living/carbon/human/proc/change_monitor,)
```

Comment:
I'll give that a try, but I'm not sure why that would change the proc's `src` to client, unless BYOND has some eldritch list declaration syntax oddities I'm not aware of